bundles/net.tourbook/src/net/tourbook/messages.properties

```diff
@@ -3461,12 +3481,12 @@ merge_tour_source_graph_heartbeat_tooltip   = Merge heartbeat values from source
 merge_tour_source_graph_temperature         = Merge &Temperature
 merge_tour_source_graph_temperature_tooltip = Merge temperature values from source into target tour
 
-pref_appearance_number_of_recent_tags         = Number of recent &tags:
+pref_appearance_number_of_recent_tags         = Number of recent &tags
 pref_appearance_number_of_recent_tags_tooltip = Number of recently used tags which are displayed in the context menu.\n\
                                                 \n\
                                                 0 will hide recently used tags.
-pref_appearance_pace_computation_option         = Pace computation options:
-pref_appearance_pace_computation_option_tooltip = This menu gives the ability to choose the time value to be used to compute and display pace values.
+pref_appearance_paceandspeed_computation_option         = Pace and Speed computation options
+pref_appearance_paceandspeed_computation_option_tooltip = This menu gives the ability to choose the time value to be used to compute and display pace and speed values.
```

Comment:
The tooltip text `This menu gives...` references a menu but there is no a menu

Reply:
`And` in a action/group label should not be written in Uppercase this was the reason why I renamed `Pace And Speed Display`

When I'm not sure to write in upper- or lower-case. I'm looking at the Eclipse UI.

![mt-label-uppercase](https://user-images.githubusercontent.com/1283445/93980041-66c34000-fd7e-11ea-8572-72026ed43e5c.png)

Reply:
Good point, I will use this tip in the future

Reply:
And I have just removed the "menu" text.